CODEOWNERS

```diff
@@ -85,6 +85,7 @@ Kconfig*                                  @tejlmand
 /samples/usb/usb_uart_bridge/             @jhn-nordic @jtguggedal
 /samples/zigbee/                          @maciekfabia @mariuszpos
 /samples/CMakeLists.txt                   @tejlmand
+/samples/nrf5340/netboot/                 @hakonfam
```

Comment:
Should we put this in the commit where it is added?

Reply:
Yes